docs/tutorial.md

```diff
@@ -1,3 +1,55 @@
 # Tutorial
 
-TODO Use demo_plugin to describe how to do stuff.
+To help illustrate the concepts of NewHELM, we provide a series of functional (if silly) examples in the `demo_plugin` folder. This tutorial walks through those examples.
+
+# Tests
+
+## Creating a basic Test
+
+[Demo: DemoSimpleQATest](../demo_plugin/newhelm/tests/demo_01_simple_qa_test.py)
+
+Let's say we want to create a Test where we send a bunch of questions to the SUT, and expect it to respond with specific answers. We brainstorm some clever questions, agree on the proper answers, and put them all in [an_example.jsonl](https://storage.googleapis.com/mlc-aisafety-scratch-dev-7292542/an_example.jsonl).
```

Comment:
This is ok for now, but the bucket name is pretty ugly. Now that the repo is public, could we move toward keeping the files in the project or in something closely aligned, like GH Pages?

Reply:
I'll use https://github.com/mlcommons/newhelm/pull/152 to retroactively fix this and the actual results.